index.js

```diff
@@ -0,0 +1,16 @@
+var fs = require('fs-extra');
+var twigger = require('pattern-importer').twigCompiler;
+
+function patternsMenu (src, obj, dest) {
```

Comment:
hey @scottnath  - should i rename this too - objectToList - or something similar?